packages/fakta-fordel-beregningsgrunnlag/src/FordelBeregningsgrunnlagFaktaIndex.jsx

```diff
@@ -25,23 +51,52 @@ const FordelBeregningsgrunnlagFaktaIndex = ({
   readOnly,
   harApneAksjonspunkter,
   submittable,
-}) => (
+}) => {
+  
+  const harFlereBeregningsgrunnlag = Array.isArray(beregningsgrunnlag);
+  const skalBrukeTabs = harFlereBeregningsgrunnlag && beregningsgrunnlag.length > 1;
+  const [aktivtBeregningsgrunnlagIndeks, setAktivtBeregningsgrunnlagIndeks] = useState(0);
+  const aktivtBeregningsrunnlag = harFlereBeregningsgrunnlag
+    ? beregningsgrunnlag[aktivtBeregningsgrunnlagIndeks]
+    : beregningsgrunnlag;
+  
+  const vilkårsperioder = behandling?.behandlingsresultat?.vilkårResultat.BEREGNINGSGRUNNLAGVILKÅR;
```

Comment:
fungerer behandling? i jsx? Trodde man måtte ha typescript for det

Reply:
Det fungerer tydeligvis.